inc/Engine/Preload/Database/Tables/Cache.php

```diff
@@ -14,6 +14,7 @@ class Cache extends Table {
 	public function __construct() {
 		parent::__construct();
 		add_action( 'rocket_preload_activation', [ $this, 'maybe_upgrade' ] );
+		add_action( 'admin_init',  [ $this, 'maybe_trigger_recreate_table' ], 9 );
```

Comment:
@CrochetFeve0251 As per discussions with @engahmeds3ed, going with the `admin_init` hook over the `init` hook was due to performance and this will run before https://github.com/wp-media/wp-rocket/blob/c5028cecd905176ddae3e2d313fb9e01aa8b9352/inc/Dependencies/Database/Table.php#L876 with this accessing the frontend also recreates the table, don't know about CRON.

Do we combine this with a guard for table related methods @piotrbak  @engahmeds3ed